### ISSUE TITLE
Make assembly names in NuGet package consistent

### ISSUE THREAD

**Issue by Redth** [micjahn/ZXing.Net#249]
This is a breaking change, but a required one for [ZXing.Net.Mobile](https://github.com/Redth/ZXing.Net.Mobile) to take a nuget dependency on ZXing.Net instead of bundling the binary directly into its own nuget package.

The problem is that for `monoandroid` (zxing.monoandroid.dll) the assembly name is different than for `netstandard` (zxing.dll).  This causes problems with bait and switch.

Could you please consider renaming the zxing.monoandroid.dll assembly?  Perhaps you could stub out zxing.monoandroid.dll to just contain `[assembly: TypeForwardedTo(..)]` attributes as to not break existing users of this package?

**Comment by micjahn** [comment]
Other question, is the zxing.monoandroid assembly really necessary? Or is a netstandard version enough?
Another possible way would be a separate binding package like that one for skiasharp, imagesharp and the others. Is it possible to mix a netstandard assembly with a specific assembly targeted against an android version?

**Comment by Redth** [comment]
Truthfully it is really not that necessary. There were some platform specific luminance sources but I end up using a custom one anyway https://github.com/Redth/ZXing.Net.Mobile/blob/master/ZXing.Net.Mobile/Android/FastJavaByteArrayYUVLuminanceSource.android.cs

There’s also a BitmapRenderer and BarcodeWriter for each platform but I include them in ZXing.Net.Mobile anyway. 

Right now I’m just referencing the netstandard2 library, and not the mono android specific one. 

Tl;dr I think you could just remove any xamarin specific implementations from your nuget.

**Comment by Redth** [comment]
Alternatively you I could make a PR to contribute the luminance sources, bitmap renderers, and barcode writers as well as change the assembly name for the platforms so this all works in your nuget.

**Comment by micjahn** [comment]
with release 0.16.6 I move out the specific mono android version to a separate binding package. Not sure if it would fix the problem, but perhaps. You can try it out. If it doesn't work, reopen this issue.

**Comment by Redth** [comment]
Thanks for this, it seems that it will do the trick!  Next update of mine will take a proper nuget package dependency on your library.  :)
